client/my-sites/upgrades/domain-management/dns/a-record.jsx

```diff
@@ -45,7 +45,7 @@ const ARecord = React.createClass( {
 					<FormLabel>{ this.translate( 'Name', { context: 'Dns Record' } ) }</FormLabel>
 					<FormTextInputWithAffixes
 						name="name"
-						placeholder={ this.translate( 'Enter subdomain (required)', { context: 'Placeholder shown when entering the required subdomain part of a new DNS record' } ) }
+						placeholder={ this.translate( 'Enter subdomain', { context: 'Placeholder shown when entering the required subdomain part of a new DNS record' } ) }
```

Comment:
I think this should be changed to `Enter subdomain (optional)` to be consistent with other types of records where the subdomain part is optional. Plus, it makes it extra clear that it is in fact optional :wink: